src/main/java/Airport.java

```diff
@@ -81,33 +116,43 @@ public PassengerPlane getPassengerPlaneWithMaxPassengersCapacity() {
         return experimentalPlanes;
     }
 
-    public Airport sortByMaxDistance() {
-        Collections.sort(planes, new Comparator<Plane>() {
-            public int compare(Plane o1, Plane o2) {
-                return o1.GetMaxFlightDistance() - o2.GetMaxFlightDistance();
-            }
-        });
-        return this;
+
+    public List<? extends Plane> sortByMaxDistance(List<? extends Plane> planes) {
+        for (Plane plane :
+                planes) {
+            Collections.sort(planes, new Comparator<Plane>() {
+                public int compare(Plane o1, Plane o2) {
+                    return o1.getMaxFlightDistance() - o2.getMaxFlightDistance();
+                }
+            });
+        }
+        return planes;
     }
 
 
-    public Airport sortByMaxSpeed() {
-        Collections.sort(planes, new Comparator<Plane>() {
-            @Override
-            public int compare(Plane o1, Plane o2) {
-                return o1.getMaxSpeed() - o2.getMaxSpeed();
-            }
-        });
-        return this;
+    public List<? extends Plane> sortByMaxSpeed(List<? extends Plane> planes) {
```

Comment:
Could you please comment on why you rewritten these comparators. I don't see a necessity to redo it like this. Do you use them in this new way?